AcceleratorLogParser.py

```diff
@@ -2,6 +2,88 @@
 '''
 Parse Altair Accelerator logfiles and write job information to a yaml file.
 
+Development notes:
```

Comment:
I moved these notes to the parser script. Customer doesn't need this in the docs.